tests/unit/plugins/modules/net_tools/test_dnsimple_info.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+from ansible_collections.community.general.plugins.modules.net_tools import dnsimple_info
+from ansible_collections.community.general.tests.unit.plugins.modules.utils import AnsibleFailJson, ModuleTestCase, set_module_args
+from ansible_collections.community.general.tests.unit.compat.mock import patch
+import pytest
+import sys
+
+
+dnsimple = pytest.importorskip('dnsimple_info')
+mandatory_py_version = pytest.mark.skipif(
+    sys.version_info < (3, 6),
+    reason='The dnsimple dependency requires python3.6 or higher'
+)
+
+
+class TestDNSimple(ModuleTestCase):
+    """Main class for testing dnsimple module."""
+
+    def setUp(self):
+        """Setup."""
+        super(TestDNSimple, self).setUp()
+        self.module = dnsimple_info
+
+    def tearDown(self):
+        """Teardown."""
+        super(TestDNSimple, self).tearDown()
+
+    def test_with_no_parameters(self):
+        """Failure must occurs when all parameters are missing"""
+        with self.assertRaises(AnsibleFailJson):
+            set_module_args({})
+            self.module.main()
+
+    def test_with_only_account_parameters(self):
+        with self.assertRaises(AnsibleFailJson):
+            set_module_args({
+                "api_key": "abcd1324",
+                "account_id": "1234"
+            })
+            self.module.main()
```

Comment:
What exactly are you testing here? The module shouldn't fail when `api_key` and `account_id` are passed (except if they are wrong). Same in the tests below.